ExRam.Gremlinq.Core/Models/Metadata/IMetadataBuilder.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace ExRam.Gremlinq.Core
+{
+    public interface IMetadataBuilder<TElement>
+    {
+        IMetadataBuilder<TElement> ReadOnly<TProperty>(Expression<Func<TElement, TProperty>> propertyExpression);
```

Comment:
What will be the difference between ReadOnly and Ignored?

Reply:
I see your point. The Enum part feels like it interrupts the clean flow of the API, so I renamed the methods to IgnoreOnUpdate and IgnoreAlways. Let me know if you still want to go the Enum route and I can make the change.